plugins/plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/navigation/filestructure/SymbolNode.java

```diff
@@ -9,40 +9,40 @@
  * Contributors:
  *   Red Hat, Inc. - initial API and implementation
  */
-package org.eclipse.che.plugin.languageserver.ide.filestructure;
+package org.eclipse.che.ide.ext.java.client.navigation.filestructure;
 
 import com.google.inject.Inject;
 import com.google.inject.assistedinject.Assisted;
 import java.util.List;
 import java.util.stream.Collectors;
 import org.eclipse.che.api.promises.client.Promise;
 import org.eclipse.che.api.promises.client.PromiseProvider;
+import org.eclipse.che.ide.ext.java.client.util.SymbolIcons;
 import org.eclipse.che.ide.ui.smartTree.data.AbstractTreeNode;
 import org.eclipse.che.ide.ui.smartTree.data.HasAction;
 import org.eclipse.che.ide.ui.smartTree.data.Node;
 import org.eclipse.che.ide.ui.smartTree.presentation.HasNewPresentation;
 import org.eclipse.che.ide.ui.smartTree.presentation.NewNodePresentation;
 import org.eclipse.che.jdt.ls.extension.api.dto.ExtendedSymbolInformation;
-import org.eclipse.che.plugin.languageserver.ide.navigation.symbol.SymbolKindHelper;
 
 /**
  * A node presenting {@link ExtendedSymbolInformation} objects
  *
  * @author Thomas Mäder
  */
 public class SymbolNode extends AbstractTreeNode implements HasNewPresentation, HasAction {
-  private SymbolKindHelper symbolHelper;
+  private SymbolIcons symbolIcons;
   private PromiseProvider promiseProvider;
   private ElementSelectionDelegate<ExtendedSymbolInformation> delegate;
   private ExtendedSymbolInformation symbol;
 
   @Inject
   public SymbolNode(
-      SymbolKindHelper symbolHelper,
+      SymbolIcons symbolHelper,
```

Comment:
I guess `SymbolIcons symbolHelper` should be replaced with `SymbolIcons symbolIcons`